packages/app/src/data-sync/indexes/index.ts

```diff
@@ -7,3 +7,4 @@ export * from "./contact.indexer"
 export * from "./message.indexer"
 export * from "./template.indexer"
 export * from "./thread.indexer"
+export * from "./phoneNumber.indexes"
```

Comment:
Please use  kebab-case (dash-case) for file names

```suggestion
export * from "./phone-number.indexes"
```